src/hotspot/share/opto/callnode.hpp

```diff
@@ -1190,9 +1190,16 @@ class AbstractLockNode: public CallNode {
 //    2 -   a FastLockNode
 //
 class LockNode : public AbstractLockNode {
+  static const TypeFunc *_lock_type_Type;
 public:
 
-  static const TypeFunc *lock_type() {
+  static inline const TypeFunc *lock_type() {
```

Comment:
Suggestion: `static inline const TypeFunc* lock_type() {`

Reply:
Sorry, that was my editor's default behaviour. Updated.